sdk/communication/communication-sms/test/smsClient.spec.ts

```diff
@@ -34,70 +36,132 @@ describe("SmsClient", async () => {
     }
   });
 
-  it("sends a SMS message to multiple recipients", async () => {
-    const connectionString = env.AZURE_COMMUNICATION_LIVETEST_CONNECTION_STRING as string;
+  //helper functions
+  const expectCorrectResult = (actualSmsResult: SmsSendResult, expectedRecipient: string) => {
+    assert.equal(actualSmsResult.httpStatusCode, 202);
+    assert.equal(actualSmsResult.to, expectedRecipient);
+    assert.isString(actualSmsResult.messageId);
+    assert.isTrue(actualSmsResult.successful);
+    assert.notExists(actualSmsResult.errorMessage, "no error message for success");
+  };
+
+  const expectIncorrectResult = (actualSmsResult: SmsSendResult, expectedRecipient: string) => {
+    assert.equal(actualSmsResult.httpStatusCode, 400);
+    assert.equal(actualSmsResult.to, expectedRecipient);
+    assert.notExists(actualSmsResult.messageId, "no message id for errors");
+    assert.isFalse(actualSmsResult.successful);
+    assert.exists(actualSmsResult.errorMessage);
+  };
+
+  it("can send a SMS message", async () => {
     const fromNumber = env.AZURE_PHONE_NUMBER as string;
     const validToNumber = env.AZURE_PHONE_NUMBER as string;
-    const invalidToNumber = "+18005551234567";
+    const results = await smsClient.send({
+      from: fromNumber,
+      to: [validToNumber],
```

Comment:
Is it possible for the recipient to just be a string?  In python, we allow that.

Reply:
we currently don't do that in JS/TS, and arch board hasn't said anything about it. @DominikMe ?